src/parkapi_sources/converters/opendata_swiss/models.py

```diff
@@ -0,0 +1,200 @@
+                check_counter_24_7 += 1
+
+            # Add opening times to fallback dict
+            open_swiss_opening_times_by_weekday[days_of_week_input].append(opening_time)
+
+            # If we have a weekday, add it to weekday list in order to check later if all weekdays have same data
+            if days_of_week_input in list(OpenDataSwissOperationTimeDaysOfWeek)[:5]:
+                check_list_weekday.append(opening_time)
+
+        # If the check counter is 7, all weekdays are open at all time, which makes it 24/7. No further handling needed in this case.
+        if check_counter_24_7 == 7:
+            return '24/7'
+
+        osm_opening_hour: list = []
+        # If all Mo-Fr entries are the same, we can summarize it to the Mo-Fr entry, otherwise we have to set it separately
+        if len(check_list_weekday) == 5 and len(set(check_list_weekday)) == 1:
+            osm_opening_hour.append(f'Mo-Fr {check_list_weekday[0]}')
+        else:
+            for weekday in list(OpenDataSwissOperationTimeDaysOfWeek)[:5]:
+                if weekday in list(open_swiss_opening_times_by_weekday):
+                    osm_opening_hour.append(
+                        f'{weekday.to_osm_opening_day_format()} {",".join(open_swiss_opening_times_by_weekday[weekday])}'
+                    )
+
+        # Weekends are handled separately
+        for weekend_day in [OpenDataSwissOperationTimeDaysOfWeek.SATURDAY, OpenDataSwissOperationTimeDaysOfWeek.SUNDAY]:
+            if weekend_day in list(open_swiss_opening_times_by_weekday):
+                osm_opening_hour.append(
+                    f'{weekend_day.to_osm_opening_day_format()} {",".join(open_swiss_opening_times_by_weekday[weekend_day])}'
+                )
+
+        return '; '.join(osm_opening_hour)
+
+
+@validataclass
+class OpenDataSwissFeatureInput:
+    id: str = StringValidator()
+    geometry: GeojsonFeatureGeometryInput = DataclassValidator(GeojsonFeatureGeometryInput)
+    properties: OpenDataSwissPropertiesInput = DataclassValidator(OpenDataSwissPropertiesInput)
+
+    def to_static_parking_site_input(self) -> StaticParkingSiteInput:
+        static_parking_site_input = StaticParkingSiteInput(
+            uid=str(self.id),
+            name=self.properties.displayName,
+            operator_name=self.properties.operator,
+            lat=self.geometry.coordinates[1],
+            lon=self.geometry.coordinates[0],
+            static_data_updated_at=datetime.now(tz=timezone.utc),
+            has_realtime_data=False,
+        )
+
+        if self.properties.parkingFacilityCategory == PurposeType.CAR:
+            static_parking_site_input.purpose = PurposeType.CAR
+            static_parking_site_input.type = ParkingSiteType.CAR_PARK
+        elif self.properties.parkingFacilityCategory == PurposeType.BIKE:
+            static_parking_site_input.purpose = PurposeType.BIKE
+            static_parking_site_input.type = ParkingSiteType.OFF_STREET_PARKING_GROUND
```

Comment:
`OFF_STREET_PARKING_GROUND` is for cars as the enum documents. There is a `BIKE_GENERIC` for unknown bike types instead.

Reply:
Now added ```GENERIC_BIKE````